src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/autocrafters/AbstractAutoCrafter.java

```diff
@@ -202,13 +202,7 @@ protected void tick(@Nonnull Block b, @Nonnull Config data) {
     protected boolean isValidInventory(@Nonnull Block block) {
         Material type = block.getType();
 
-        // TODO: Add designated SlimefunTag
-        return switch (type) {
-            case CHEST,
-                TRAPPED_CHEST,
-                BARREL -> true;
-            default -> SlimefunTag.SHULKER_BOXES.isTagged(type);
-        };
+        return SlimefunTag.STORAGE_BLOCKS.isTagged(type);
```

Comment:
This tag does not necessarily represent all storage blocks. Only the ones Auto Crafters support. The tag name should reflect that.